weave/trace_server/migrations/022_add_annotation_queues.up.sql

```diff
@@ -0,0 +1,266 @@
+/*
+This migration creates the queue-based call annotation system tables.
+
+The system consists of three main tables:
+1. annotation_queues: Queue definitions with metadata and scorers
+2. annotation_queue_items: Tracks which calls belong to each queue (membership)
+3. annotator_queue_items_progress: Tracks per-annotator workflow state for each queue item
+
+Key features:
+- Lightweight UPDATE support via enable_block_number_column
+- Multi-queue support (same call can be in multiple queues)
+- Multi-annotator support with work claiming and expiration
+- Display field selection for focused annotation
+*/
+
+-- ============================================================================
+-- annotation_queues: Queue definitions and metadata
+-- ============================================================================
+CREATE TABLE annotation_queues (
```

Comment:
this is strikingly similar to monitors (a name + scorers) - i am curious if you explored using the same structure as monitors.